crates/turborepo-ui/src/tui/app.rs

```diff
@@ -126,6 +131,18 @@ impl<W> App<W> {
         })
     }
 
+    fn update_sidebar_toggle(&mut self) {
```

Comment:
I feel like toggle implies an update, up to you if this is better name 🤷 
```suggestion
    fn toggle_sidebar(&mut self) {
```